doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(DLE))`
+
+Data per item is:
+
+* `D`: PSKd
+* `L`: Timeout in seconds
+* `E`: Extended/long address (optional)
+
+Passess Extended address and PSKd to the NCP in the commissioning process.
```

Comment:
Need to explicitly document that the extended address should be omitted if not used.